README.md

```diff
@@ -177,44 +177,64 @@ Example of a partial CR:
 
 ### Mounting and injecting binding information
 
-This specification allows for data to be mounted using volumes or injected using environment variables.  The best practice is to mount any sensitive information, such as passwords, since that will avoid accidentally exposure via environment dumps and subprocesses.  Also, binding binary data (e.g. .p12 certificate for Kafka) as an environment variable might cause a pod to fail to start (stuck on `CrashLoopBackOff`), so it advisable for backing services with such binding data to mark it with `bindAs: volume`.
+#### Mounting data
+Implementations of this specification **MUST** mount the binding data into the consuming application container in the following location:
+
+```
+<mountPathPrefix>/<service-id>/secret/<persisted_secret>
+```
+
+Where:
+* `<mountPathPrefix>` defaults to `/platform/bindings` if not specified in the `ServiceBinding` CR via the `mountPathPrefix` element.  The environment variable `SERVICE_BINDINGS_ROOT` **MUST** be set to the value of `<mountPathPrefix>`, so that applications can always find this directory, and it is immutable.  This means that if another `ServiceBinding` CR wants to project itself into the same container, it **MUST** reuse the current `SERVICE_BINDINGS_ROOT` value.  
```

Comment:
I'd be very careful defining a default value for this (we got some pushback on it), but if you do I'd also choose something shallower.  The CNB project chose `/platform/bindings` (and only at build time, not run time) because there are so many moving parts in the spec, that it made sense to have an umbrella directory where `bindings` is a peer to other kinds of data.  Because this spec is concerned with a single data type, bindings, the default (if it exists) should probably be a top-level directory, `/bindings`.

Reply:
Also, I believe that both having a default and requiring `$SERVICE_BINDINGS_ROOT` are in opposition to one another.  If the environment variable is always required, then no default should be necessary and vice-versa.

Reply:
That's a good point about `/bindings`.  I must say that `/platform/bindings` was heavily influenced by the desire to stay consistent with what CNB had done.  

The `$SERVICE_BINDINGS_ROOT` is an interesting one:   from the app perspective, it can rely that it will always be set.   So the platform must always set it, even when the author of the binding did not specify a `mountPathPrefix`, thus the platform would need a default to use in such cases.

Reply:
The default should only exist within the reconciler attaching the binding to the workload. The env var should always be defined for a container that has any service bindings. In short, a workload can define the var to instruct the bindings to be mounted in a specific location, otherwise the first reconciler to bind into the workload gets to define the path (to the default value).